### ISSUE TITLE
Question: Does this plugin depends on gradle android plugin version?

### ISSUE THREAD

**Issue by paulvi** [shwenzhang/AndResGuard#389]
Does this plugin depends on gradle android plugin version?

'com.android.tools.build:gradle:3.5.1'

By default this version is the same as Android Studio version.

Can there be issue with older build tools version or SDK build toolds version?

What are minimum gradle android plugin and SDK build tools version?

**Comment by paulvi** [comment]
Build Tools 26.0.2 or higher for  v3.0.0
Build Tools 27.0.3 or higher for  v3.1.0
Build Tools 28.0.3 or higher for  v3.2.0-3.5.x

Newer than default Build Tools version can be specified,
but older will be ignored and overridden with default, e.g.

> WARNING: The specified Android SDK Build Tools version (26.0.2) is ignored, as it is below the minimum supported version (28.0.3) for Android Gradle Plugin 3.5.1.
Android SDK Build Tools 28.0.3 will be used.

**Comment by simpleton** [comment]
Good point! Ideally, AndResGuard should work on most version of android gradle plugins. But it's good to have this compatible list.

**Comment by paulvi** [comment]
Sure, older version may not work with newer Android plugin/ SDK build tools.  
At the same time current (latest and the best) version of the AndResGuard may not support absolutely all Android plugin/ SDK build tools.  

So it should be very clearly specified (in README),  
at least what is known by authors.
